File: server/src/main/java/com/example/server/model/People.java
package com.example.server.model;

import com.example.server.model.common.BaseEntity;
import org.babyfish.jimmer.sql.*;


@Entity
@Table(name = "people")
public interface People extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    long id();

    String username();

    String password();

    Integer roleId();

    @ManyToOne
    Role role();
}

File: website/src/__generated/model/entities/People.ts
import type { Role } from './';

export interface People {
    
    readonly createdTime: string;
    
    readonly modifiedTime: string;
    
    readonly id: number;
    
    readonly username: string;
    
    readonly password: string;
    
    readonly roleId: number;
    
    readonly role: Role;
}
